net-misc/cgminer/cgminer-4.11.1.ebuild

```diff
@@ -1,20 +1,21 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
 inherit flag-o-matic toolchain-funcs udev
 
 DESCRIPTION="Bitcoin CPU/GPU/FPGA/ASIC miner in C"
 HOMEPAGE="https://bitcointalk.org/?topic=28402.msg357369 https://github.com/ckolivas/cgminer"
-SRC_URI="http://ck.kolivas.org/apps/cgminer/${P}.tar.bz2"
-#SRC_URI="http://ck.kolivas.org/apps/cgminer/4.5/${P}.tar.bz2"
+SRC_URI="https://github.com/ckolivas/cgminer/releases/tag/v${PV}"
```

Comment:
did you mean?
SRC_URI="https://github.com/ckolivas/cgminer/archive/v${PV}.tar.gz -> ${P}.tar.gz"